service/templates/task.html

```diff
@@ -344,10 +344,16 @@ <h3 class="section-heading"><i class="fa fa-exchange"></i> Depends On</h3>
           </div>
           <div class="col-lg-5">
             <div class="row">
-              <div class="col-md-4">
+              <div class="col-md-3">
                 <h3 class="section-heading"><i class="fa fa-flask"></i> Tests</h3>
               </div>
-              <div class="col-md-8">
+              <div class="col-md-5">
+                <div ng-hide="!task.test_results || task.test_results.length == 0"
```

Comment:
In general you should use === and !== for comparisons. Using == means that anything that can be type converted to 0 (such as the string "0") will return true. We aren't consistent about it in existing code, but we should do it for new code

Reply:
Fixed